Strategy11/formidable-forms
# Compliance with White Labeling in Backend

This PR addresses an oversight in the backend white labeling, particularly in the import menu. It ensures that when white labeling is enabled, the user-chosen brand name replaces 'Formidable' in all backend interfaces, including the import page.

## Related Issue:
[White labeling needs to comply in the backend](https://github.com/Strategy11/formidable-pro/issues/4058)

## QA URL
https://qa.formidableforms.com/sherv11/wp-admin/admin.php?page=formidable-import

## Testing Instruction:
1. Navigate to `WP Admin > Formidable > Global Settings > White Labeling`.
2. Change the 'Plugin Label' to a value different from 'Formidable'.
3. Go to `WP Admin > Formidable > Import/Export`.
4. Confirm that 'Formidable' is replaced with the new brand name in the page descriptions and the XML file title section.
5. > Important: After changing the brand name, ensure you `disable` the` Formidable PRO` plugin. Note that there is a separate PR for the PRO version.
https://github.com/Strategy11/formidable-pro/pull/4559

## Output:
### Before
<kbd>
<img width="1221" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/12138515-ecc8-4aa9-8f3d-141ef0968593">
</kbd>

---

<kbd>
<img width="1481" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/416ba294-59c8-4b6f-834f-a849f1413020">
</kbd>

### After
<kbd>
<img width="1221" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/12138515-ecc8-4aa9-8f3d-141ef0968593">
</kbd>

---

<kbd>
<img width="1501" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/cd049c7e-0581-446d-b9b9-cd76ba035f15">
</kbd>
